video/out/wayland_common.c

```diff
@@ -1240,6 +1241,10 @@ static void registry_handle_add(void *data, struct wl_registry *reg, uint32_t id
     if (!strcmp(interface, wp_content_type_manager_v1_interface.name) && found++) {
         wl->content_type_manager = wl_registry_bind(reg, id, &wp_content_type_manager_v1_interface, 1);
     }
+
+    if (!strcmp(interface, wp_single_pixel_buffer_manager_v1_interface.name) && found++) {
+        wl->single_pixel_manager = wl_registry_bind(reg, id, &wp_single_pixel_buffer_manager_v1_interface, 1);
+    }
```

Comment:
Could you also add logging for when this isn't found in `preinit`? i.e. the `Compositor doesn't support the %s protocol!` stuff.